ministryofjustice/laa-apply-for-criminal-legal-aid
# Add new case model

+ route namespace
+ blank decision tree

## Description of change
- just some backend code to enable the future work with case details.

## Link to relevant ticket
Part of [CRIMAP-74](https://dsdmoj.atlassian.net/browse/CRIMAP-74)

## Notes for reviewer
- I haven't added any tests really as this is basically just placeholder stuff.
- is there any plumbing that we need that you think would make sense here?

## How to manually test the feature
`rails routes | grep urn` - gives you the new routes to check

in the rails console you can add cases to an application now:

```ruby
c = Case.new(urn: '12345')
ca = CrimeApplication.new(case: c)

# then you can do
ca.case
>> # returns related case

c.crime_application
>> # returns crime application
```
